pkg/dmesg/events_watcher.go

```diff
@@ -0,0 +1,103 @@
+package dmesg
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/leptonai/gpud/components"
+	"github.com/leptonai/gpud/pkg/common"
+	events_db "github.com/leptonai/gpud/pkg/events-db"
+	"github.com/leptonai/gpud/pkg/log"
+)
+
+type eventsWatcher struct {
```

Comment:
We already have a type named `Watcher`, so we should not call the new type `eventsWatcher`. We can call it `LogLineProcessor`, `LogEventProcessor`, `LogLineSink`?

Reply:
@eahydra Renamed